indico/modules/events/views.py

```diff
@@ -120,9 +120,43 @@ def _getHeader(self):
     def getJSFiles(self):
         return WPDecorated.getJSFiles(self) + self._asset_env['modules_event_display_js'].urls()
 
+    def _getJsonLDData(self):
+        lddata = {
+            "@context": "http://schema.org",
+            "@type": "Event",
+            "url": self.event.external_url,
+            "name": self.event.title,
+            "startDate": self.event.start_dt.isoformat(),
+            "endDate": self.event.end_dt.isoformat(),
+            "location": {
+                "@type": "Place",
+                "name": self.event.venue_name or 'No location set',
```

Comment:
Is this required? Otherwise we could maybe just omit the location if there's no location information and/or use an empty string in case of just one of the two pieces of data being empty. IMO an empty string is more meaningful than a custom string where only a human knows that it means "not set/known"

Reply:
At least the google tester flags this a errors, and an empty string is also not allowed.